src/request.js

```diff
@@ -15,6 +15,7 @@ import nameFor from './naming';
 
 const DEFAULT_TIMEOUT = 5000;
 const DEFAULT_RETRIES = 0;
+const DEFAULT_PROXY_RETRIES = 3;
```

Comment:
I am defaulting everywhere to retry proxy errors 3 times. Let me know if I should set this to 0 and enable retries only for Styx

Reply:
We should be able to configure all the way from the container level using env vars like we can with other `nodule-*` libraries. So default is 0 and then in ansible we can adjust to be 3

Reply:
also, yeah, let's just start with styx for now

Reply:
Yeah this is configurable at container level. I will default this to 0